src/parser/csql_grammar.y

```diff
@@ -3495,22 +3502,22 @@ alter_stmt
 			  }
 
 		DBG_PRINT}}
-	| ALTER						/* 1 */
-		{					/* 2 */
+	| ALTER					/* 1 */
+		{				/* 2 */
 			PT_NODE* node = parser_new_node(this_parser, PT_ALTER_INDEX);
 			parser_push_hint_node(node);
 		}
-	  opt_hint_list					/* 3 */
-	  opt_reverse					/* 4 */
-	  opt_unique					/* 5 */
-	  INDEX						/* 6 */
-	  identifier					/* 7 */
-	  ON_						/* 8 */
-	  only_class_name				/* 9 */
-	  opt_index_column_name_list			/* 10 */
-	  opt_where_clause				/* 11 */
+	  opt_hint_list				/* 3 */
+	  opt_reverse				/* 4 */
+	  opt_unique				/* 5 */
+	  INDEX					/* 6 */
+	  identifier				/* 7 */
+	  ON_					/* 8 */
+	  only_class_name			/* 9 */
```

Comment:
From the next review, please resolve the indentation issue before requesting a source code review.

Reply:
Would it be a problem to turn on the hide whitespace option and review?

Reply:
Highlighting the whole code context is a problem.